metadata.md

```diff
@@ -3,15 +3,25 @@ layout: default
 css_id: metadata
 ---
 
-# Metadata
+# Roles and Metadata
 
-Metadata files provide information that clients can use to make update decisions. Different metadata files provide different information. The various metadata files are signed by different keys as are indicated by the root role. The concept of roles allows TUF to only trust information that a role is trusted to provide.
+TUF uses metadata to create verifiable records about the state
```

Comment:
Can we begin the paragraph with a discussion on roles, so that it matches its appearance in the section header?   Or you can switch their order in the header.

Reply:
fixed.